tests/jobs/rft/test_trajectory.py

```diff
@@ -93,7 +97,7 @@ def initdir(tmpdir):
 @pytest.mark.parametrize(
     "fname", ["valid_trajectories.txt", "valid_trajectories_with_comments.txt"]
 )
-def test_load(initdir, fname):
+def test_load_and_to_dataframe(initdir, fname):
     expected_utmxs = [0, 4]
     trajectory = Trajectory.load_from_file(fname)
```

Comment:
This could be mocked for most of the tests so you don´t have to load from file each time.